NedyUdombat/ReachHigh
# Setup Express app

#### What does this PR do?
Setup Express app

#### Description of Task to be completed?
- [x] Install required dependencies
- [x] Setup entry point into application

#### How should this be manually tested? 
* clone this repo
* checkout to `ch-setup-express-app-169169088` branch
* run `npm install`
* Visit `http://127.0.0.1:8000` on your REST API client example Postman
* You should see this welcome message 
```json
{
    "message": "Welcome to ReachHigh!"
}
```
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?(if applicable)
[Pivotal tracker ID](https://www.pivotaltracker.com/story/show/169169088/)

#### Screenshots
